react/scripts/move-built.js

```diff
@@ -9,31 +9,37 @@ fsExtra.ensureDir("../assets/js/");
 // remove any existing generated files from this HUGO dir, though in prod it should be clean
 var files = globSync("../assets/js/*.{js,js.map}");
 files.forEach((file) => {
-  console.log(`file ${file}`)
-  if (file.endsWith('chunk.js') || file.endsWith('chunk.js.map') || file.startsWith('../assets/js/runtime-main')) {
-    console.log(`removing ${file}`)
-    fsExtra.removeSync(file);
-  }
+  console.log(`removing ${file}`)
+  fsExtra.removeSync(file);
```

Comment:
the files are no longer suffixed with `.chunk` so this was erroring. I decided to make it just clean the whole directory instead of only removing certain files. Am I missing something or should this be ok?

Reply:
I don't recall, should be ok!